spec/services/preserved_object_handler_create_spec.rb

```diff
@@ -40,13 +40,14 @@
       it 'logs an error' do
         po_handler.create
         expect(Rails.logger).to receive(:log).with(Logger::ERROR, exp_msg)
-        po_handler.create
+        new_po_handler = described_class.new(druid, incoming_version, incoming_size, ep)
+        new_po_handler.create
```

Comment:
this is the one test I had to change -- it's abnormal to re-use a pohandler object so I changed this test so it no longer did.

Reply:
as a stretch-goal/patch-party type thing, maybe we could be nice to the caller and set a flag internally that'd let us raise an error when an instance has been used and a new one has to be instantiated?

Reply:
sure, could be a patch party ticket.  Make it so.  Also make a case for it in the ticket??

Reply:
see #388